Bug: vite fails to resolve correctly @beeq/core/dist/* paths

### BEEQ package

beeq/react

### BEEQ version

1.4.0

### Current Behavior

✘ [ERROR] Missing "./dist/components/bq-accordion.js" specifier in "@beeq/core" package [plugin vite:dep-pre-bundle]

    node_modules/@beeq/react/src/components.js:5:57:
      5 │ ...ineBqAccordion } from '@beeq/core/dist/components/bq-accordion.js';
        ╵                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  This error came from the "onResolve" callback registered here:

    node_modules/esbuild/lib/main.js:1293:20:
      1293 │       let promise = setup({


### Expected Behavior

Should not fail.

### Steps to Reproduce

1. npm create vite@latest
2. Pick react  with typescript with swc
3. cd project && npm i
4. npm i @beeq/{core,react}
5. add this to App.tsx
```jsx
import './App.css'
import { BqButton } from '@beeq/react'

function App() {
  return <BqButton />
}

export default App

```


### Code Reproduction URL

_No response_

### Your Environment

_No response_

### Additional Information

_No response_

--- a/packages/beeq-angular/package.json
+++ b/packages/beeq-angular/package.json
@@ -17,9 +17,5 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Endava/BEEQ"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org"
   }
 }

--- a/packages/beeq-react/package.json
+++ b/packages/beeq-react/package.json
@@ -17,9 +17,5 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Endava/BEEQ"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org"
   }
 }

--- a/packages/beeq-tailwindcss/package.json
+++ b/packages/beeq-tailwindcss/package.json
@@ -13,9 +13,5 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Endava/BEEQ"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org"
   }
 }

--- a/packages/beeq-vue/package.json
+++ b/packages/beeq-vue/package.json
@@ -15,9 +15,5 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Endava/BEEQ"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org"
   }
 }

--- a/packages/beeq/README.md
+++ b/packages/beeq/README.md
@@ -12,12 +12,6 @@ BEEQ elements, components, patterns, utilities, etc., are available as an npm pa
 $ npm i @beeq/core
 ```
 
-### For `beta` releases
-
-```bash
-$ npm i @beeq/core@beta
-```
-
 ## Usage
 
 BEEQ components are regular HTML elements, or custom elements (often referred to as "web components"). If you're using a simple HTML page, you can use them like any other element.

--- a/packages/beeq/package.json
+++ b/packages/beeq/package.json
@@ -9,17 +9,6 @@
   "collection": "./dist/collection/collection-manifest.json",
   "collection:main": "./dist/collection/index.js",
   "unpkg": "./dist/beeq/beeq.esm.js",
-  "exports": {
-    ".": {
-      "import": "./dist/beeq/beeq.esm.js",
-      "require": "./dist/cjs/beeq.cjs.js"
-    },
-    "./loader": {
-      "import": "./dist/loader/index.js",
-      "require": "./dist/loader/index.cjs",
-      "types": "./dist/loader/index.d.ts"
-    }
-  },
   "files": [
     "dist/"
   ],
@@ -32,9 +21,5 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Endava/BEEQ"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org"
   }
 }

